src/com/maddyhome/idea/vim/group/ChangeGroup.java

```diff
@@ -151,10 +162,7 @@ public void insertLineStart(@NotNull Editor editor, @NotNull Caret caret, @NotNu
    * @param context The data context
    */
   public void insertAfterCursor(@NotNull Editor editor, @NotNull DataContext context) {
-    // TODO: Add multiple carets support
-    MotionGroup.moveCaret(editor, editor.getCaretModel().getPrimaryCaret(), VimPlugin.getMotion()
-      .moveCaretHorizontal(editor, editor.getCaretModel().getPrimaryCaret(), 1, true));
-    initInsert(editor, context, CommandState.Mode.INSERT);
+    insertAfterCursor(editor, editor.getCaretModel().getPrimaryCaret());
```

Comment:
This method is no longer used. Let's remove it and all the other unused methods.